account_payment_notification/data/mail_template.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<data noupdate="1">
+
+    <record id="mail_template_notification" model="mail.template">
+        <field name="name">Payment notification</field>
+        <field name="auto_delete" eval="False" />
+        <field name="lang">{{ object.partner_id.lang }}</field>
+        <field name="model_id" ref="account.model_account_payment" />
+        <field name="partner_to">{{ object.partner_id.id }}</field>
+        <field
+            name="subject"
+        >{{ object.company_id.name }} Payment Notification (Ref {{ object.name or 'n/a' }})</field>
+        <field name="body_html" type="html">
+            <p>Dear <t t-out="object.partner_id.name or ''">Azure Interior</t>,</p>
+            <p t-if="object.payment_type == 'inbound'">
+                Your debit order was processed and sent to the bank.
+            </p>
+            <p t-else="">
+                Your payment order was processed and sent to the bank.
+            </p>
+            <p t-if="object.reconciled_invoices_type == 'invoice'">
+                It includes the following invoices:
+            </p>
+            <p t-else="">
+                It includes the following credit notes:
+            </p>
+            <ul>
+                <t
+                    t-foreach="object.reconciled_invoice_ids | object.reconciled_bill_ids"
+                    t-as="invoice"
+                >
+                    <li t-out="invoice.ref or invoice.name">INV/2022/00001</li>
```

Comment:
Hello @yajo , we have here an issue that we should fix:

- Customer invoices that we debit the number of the invoice is field `name` in model `account.move`, it is not `ref`, never. And field name is mandatory so no needed an "OR"

- Supplier invoices that we credit the number of the invoice is field `ref` in model `account.move`, and name shouldn't be printed because supplier don't know what is that number. Here field ref is not mandatory, so you can say `ref` OR `payment_reference`

So, we need another **if** `inbound` then `name` , **else** `ref` OR `payment_reference`

Thanks